### ISSUE TITLE
Test failure in strict Latin-1* locale

### ISSUE THREAD

**Issue by gadenbuie** [rstudio/learnr#748]
## From CRAN

> Dear maintainer,
> 
> Please see the problems shown on
> <https://cran.r-project.org/web/checks/check_results_learnr.html>.
> 
> Apparently your package fails its checks in a strict Latin-1* locale,
> e.g. under Linux using LANG=en_US.iso88591 (see the debian-clang
> results).
> 
> Please correct before 2022-11-16 to safely retain your package on CRAN.
> 
> Best,
> -k

## From CRAN checks

https://web.archive.org/web/20221102200837/https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-clang/learnr-00check.html

```
  Running 'testthat.R' [59s/71s]
Running the tests in 'tests/testthat.R' failed.
Complete output:
  >
  > if (requireNamespace("testthat")) {
  + library(testthat)
  + library(learnr)
  +
  + test_check("learnr")
  + }
  Loading required namespace: testthat
  [ FAIL 1 | WARN 0 | SKIP 18 | PASS 812 ]
  
  == Skipped tests ===============================================================
  * On CRAN (18)
  
  == Failed tests ================================================================
  -- Failure ('test-exercise.R:1223'): evaluate_exercise() does not return a message for parsable non-ASCII code --
  result$feedback is not NULL
  
  `actual` is a list
  `expected` is NULL
  
  [ FAIL 1 | WARN 0 | SKIP 18 | PASS 812 ]
  Error: Test failures
  Execution halted
```

**Comment by rossellhayes** [comment]
I've seen this before in `and`, I think I should be able to fix it.

**Comment by gadenbuie** [comment]
Fixed in #749

**Comment by gadenbuie** [comment]
http://web.archive.org/web/20221114165247/https://cran.r-project.org/web/checks/check_results_learnr.html

![image](https://user-images.githubusercontent.com/5420529/201718499-6eb3251a-0ddf-4969-a82e-3aa6de7f3612.png)

**Comment by gadenbuie** [comment]
I don't think we need to fix those notes (but if we do, I'm sure we'll hear about it). It's because our help documentation links to help topics (functions) in packages that aren't part of our dependencies. i.e. learnr doesn't directly depend on `sortable` or `bslib`.

**Comment by schloerke** [comment]
What about hard linking to their websites?
